test/WebJobs.Script.Tests/TestScripts/CSharp/TimeoutStopHost/run.csx

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Microsoft.Azure.WebJobs.Host;
+
+public static void Run(string input, TextWriter writer, TraceWriter log)
+{
+    log.Info(input);
+
+    while (true)
+    {
+        log.Info("Waiting");
+        Thread.Sleep(100);
+    }
+}
+
+public class Context
```

Comment:
Removed this test as it wasn't interesting with the move of the logic to core.

---
In reply to: [76699772](https://github.com/Azure/azure-webjobs-sdk-script/pull/604#discussion_r76699772) [](ancestors = 76699772)